lib/error.js

```diff
@@ -4,14 +4,12 @@ var util = require('util'),
     Controller = require('hmpo-form-controller');
 
 function getArgs(type, args) {
-    if (type === 'maxlength') {
-        return { maxlength: args[0] };
-    } else if (type === 'minlength') {
-        return { minlength: args[0] };
-    } else if (type === 'exactlength') {
-        return { exactlength: args[0] };
-    } else if (type === 'past') {
+    if (type === 'past') {
         return { age: args.join(' ') };
+    } else if (args !== undefined) {
```

Comment:
Thanks for looking Daniel, good point about validation.

Looks like the form controller sets a default https://github.com/UKHomeOffice/passports-form-controller/blob/master/lib/error.js#L5 but doesn't go beyond that. I think it makes sense for any validation to go here - @gavboulton what do you think?